src/components/aboutUs/StorySection.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+//MUI styles
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+const StorySection = () => {
+    return (
+        <>
+            <Box sx={styles.bgSection}>
+                <Typography sx={styles.bgTitle}>Helping each other</Typography>
+                <Typography sx={styles.bgText}>/ DEV PREP /</Typography>
+            </Box>
+            <Box>
+                <Typography sx={styles.storyTitle}>OurStory</Typography>
+                <Typography sx={styles.storyText}>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididunt ut labore et dolore magna
+                    aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                    ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                    Duis aute irure dolor in reprehenderit in voluptate velit
+                    esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+                    occaecat cupidatat non proident, sunt in culpa qui officia
+                    deserunt mollit anim id est laborum.
+                </Typography>
+            </Box>
+        </>
+    );
+};
+
+export default StorySection;
+
+const styles = {
+    bgSection: {
+        width: "100%",
+        height: "250px",
+        bgcolor: "#e8e8e8",
```

Comment:
Again, colours should be defined in the theme and use the theme variables: Once the colours are defined in the theme there are two ways to access them in the styles object.
1. using the theme variable. requires const theme = useTheme() in the component. Then in the styles obj: 
```
const styles = {
    bgColor: {
        backgroundColor: theme.palette.primary (...or some other colour)
    }
}
```
docs: https://mui.com/material-ui/customization/theming/#accessing-the-theme-in-a-component
2. The easier option is to reference the color directly:
First create the color in the theme (this is the same for both options): https://mui.com/material-ui/customization/palette/
Then use it in the styles obj:
```
const styles = {
    bgColor: {
        backgroundColor: "primary" (...or some other colour)
    }
}
```
docs: https://mui.com/material-ui/customization/how-to-customize/#1-one-off-customization